.ci_support/linux_64_cuda_compiler_version10.0cudnn7python3.6.____73_pypy.yaml

```diff
@@ -7,7 +7,7 @@ cdt_name:
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge rc_cupy
```

Comment:
```suggestion
- conda-forge cupy_rc
```

Reply:
Oops 🙊 Thanks for catching. Will update 🙂

Reply:
Should be fixed now. We changed the CFEP since I last read it. So forgot about this.

Reply:
Thanks, John! I didn't realize until now that I commented on the yamls for CI (was on my cell) 😂